srcpkgs/hfsutils/template

```diff
@@ -0,0 +1,47 @@
+# Template file for hfsutils
+
+pkgname="hfsutils"
+version="3.2.6"
+revision=1
+build_style=gnu-configure
+short_desc="A comprehensive software package to permit manipulation of HFS volumes"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2"
+homepage="https://www.mars.org/home/rob/proj/hfs/"
+distfiles="ftp://ftp.mars.org/pub/hfs/hfsutils-${version}.tar.gz"
```

Comment:
do not use ftp if posslbe

Reply:
hfsutils is only officially available via ftp unless you want the debian re-package of it.